stix2/constants.py

```diff
@@ -0,0 +1,135 @@
+import base64
```

Comment:
I think rolling up to stix2.patterns is the right level...

I will make this change.